src/connection.js

```diff
@@ -36,7 +36,8 @@ module.exports = function (classes){
           id = ++self.latestId;
 
           self.callbacks[id] = {
-            resolve, reject
+            'resolve': resolve, 

```

Comment:
forgot that it needs to be ES5 compatible heh https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Operators/Object_initializer#New_notations_in_ECMAScript_2015